packages/did-provider-ebsi/src/services/EbsiRPCService.ts

```diff
@@ -0,0 +1,157 @@
+import { Headers } from 'cross-fetch'
+import {
+  AddVerificationMethodParams,
+  AddVerificationMethodRelationshipParams,
+  GetDidDocumentParams,
+  GetDidDocumentsParams,
+  InsertDidDocumentParams,
+  SendSignedTransactionParams,
+  UpdateBaseDocumentParams,
+  Response,
+  GetDidDocumentsResponse,
+} from '../types'
+import { DIDDocument } from 'did-resolver'
+
+/**
+ * @constant {string} jsonrpc
+ */
+const jsonrpc = '2.0' // optional param of plugin?
+const baseUrl = 'https://api-pilot.ebsi.eu/did-registry/v5/jsonrpc' // optional param of plugin?
```

Comment:
This should be part of a string union containing the different EBSI environments. Then we should concatenate the version and did-registry.

Reply:
Created EbsiEnvironment type which is a string intersection of different environments and moved it to `src/types.ts`. Created a function `getDidRegistryRPCUrl` to build the url and moved it to `src/function.ts`. Adjusted the service methods and the plugin constructor to receive an additional optional parameter: apiOpts